src/containers/SignUpForm/SignUpForm.js

```diff
@@ -55,7 +55,7 @@ export class SignUpForm extends Component {
           required
           className={inputStyle}
         />
-        {!this.checkEmailRegex(email) && <p>Enter a valid email</p>}
+        {!this.checkEmailRegex(email) && <p className="notice">Please enter a valid email.</p>}
```

Comment:
We have to figure out a way so that when this message is not showing, the space it would take up is still there. Right now as the error pops up and goes away the spacing of the elements changes.